content/includes/nap-waf/config/common/ip-groups-override-rules.md

```diff
@@ -0,0 +1,72 @@
+#### IP-Groups feature as part of Override Rules feature.
+The Override Rules feature allows you to modify original or parent policy settings.
```

Comment:
It's a markdown standard to include a blank line between headers and text (though it still works without)

```suggestion

The Override Rules feature allows you to modify original or parent policy settings.
```